website/docs/cloud-docs/users-teams-organizations/single-sign-on/okta.mdx

```diff
@@ -27,6 +27,9 @@ For more information on the listed features, visit the [Okta Glossary](https://h
 
 For information on configuring automated team mapping using Okta group membership, please see the [Team Mapping Configuration (Okta)](#team-mapping-configuration-okta-) section below.
 
+Creation of an Application on Okta side requires elevated privileges. For that reason customers sometimes create a workflow to provision it, leaving the end user with a need to configure the actual settings. 
```

Comment:
Because this is a very particular scenario you are describing about setting up an Okta feature, rather than an HCP Terraform feature, it feels like it might be a better fit for a support article, rather than an additional to our documentation. In general, we want to try to avoid describing how other products work in our documentation as much as possible. Let me know what you think though! Happy to have a conversation about this if you disagree!

Reply:
I understand. Let me rephrase a little bit then.
Configuration using Terraform Cloud application from Okta side is flawless and requires very few steps.
Some customers don't want to grant this level of access and they use a separate workflow to create a generic app in Okta. When you look at the table from step 4, Single sign on URL on our side is ACS Consumer in Okta. Customers tend to paste a wrong value in Okta because our config shows Single sign on URL as well.
To sum it up, they do not what's in the table, but Single sign on from HCP -> Single sign on in Okta.
This already exists in TFE docs, probably for exactly that reason. I can do the article if you still think it's the best route. Just wanted to make sure the why is clear.

Reply:
Thank you for the extra context, I really appreciate it! I do still think this would be better presented in a support article because it is such a particular use case and workflow that has more to do with Okta than on our own products. Thanks for looping me in though!